.github/draft_release_notes.sh

```diff
@@ -0,0 +1,54 @@
+#! /bin/sh
+
+# this script assumes the GITHUB_TOKEN and PREVIOUS_TAG environment variables have been set;
+# it produces a 'Changes.md' file as its final output;
+# the file 'last-300-prs-with-release-note.txt' that is produces is intermediate data; it is not
+# pruned for now to assist development of the release notes process (we are still curating all this)
+
+sudo apt-get -y install jq
+sudo apt-get -y install wget
+curl -L https://github.com/github/hub/releases/download/v2.14.2/hub-linux-amd64-2.14.2.tgz | tar xzf -
+PWD=`pwd`
+export PATH=$PWD/hub-linux-amd64-2.14.2/bin:$PATH
+git fetch --all --tags --prune --force
+echo -e "# Insert Title\n" > Changes.md
+echo -e "## Features\n\n## Fixes\n\n## Backwards incompatible changes\n\n## Docs\n\n## Misc\n\n## Thanks" >> Changes.md    - name: Draft Release
```

Comment:
Did a [`shellcheck`](https://github.com/koalaman/shellcheck) run and it tells me if we run with `/bin/sh` then `echo` does not support flags. However, I tried it locally and it works. Not sure about this one. Maybe we switch to a HEREDOC instead since we have two `echo`s  already.

```suggestion
cat <<EOF >Changes.md
# Insert Title

## Features

## Fixes

## Backwards incompatible changes

## Docs

## Misc

## Thanks

EOF
```


This aside, I am wondering about the trailing `- name: Draft Release` at the end of the line. Should that be removed?

Reply:
sure I'll switch to cat / HEREDOC in the next push.

per prior discussion with @qu1queee @HeavyWombat  I think we want to leave the `Draft Release` disclaimer at least for a bit while we iterate on this whole notion, and allow for manual tweaks before making them official.  Where eventually we might become comfortable enough with the automation that we then remove the draft notion.

Reply:
actually now that I think about it ... when I add the support for processing the kind labels, I think I'm going to handle these headers iteratively as I process each kind, so I'm going to land back with multiple echo's anyway.

So I'm going to leave this as is for now @HeavyWombat - thanks